php/Terminus/Models/Collections/Workflows.php

```diff
@@ -76,11 +76,10 @@ protected function getFetchUrl() {
         );
           break;
       case 'organization':
-        $user = $this->get('user');
         $url  = sprintf(
           'users/%s/organizations/%s/workflows',
-          $user->id,
-          $this->owner->get('id')
+          $this->owner->user->id,
+          $this->owner->organization->id
```

Comment:
I don't think this will work, but it may not have ever worked. I think the user_id should be the ID of the session user (the person operating terminus) and the Owner is the Org.